tests/helpers.test.ts

```diff
@@ -0,0 +1,115 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Neil Enns. All rights reserved.
+ *  Licensed under the MIT License. See LICENSE in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+import {closeSync, existsSync, openSync, unlinkSync, writeFileSync} from 'fs';
+import * as JSONC from "jsonc-parser";
+import * as helpers from "./../src/helpers";
+
+describe("helpers", () => {
+  const settingsFilePath = `${__dirname}/settings.json`;
+
+  beforeEach(() => {
+    closeSync(openSync(settingsFilePath, 'w'));
+  });
+
+  afterEach(() => {
+    jest.clearAllMocks();
+    if (existsSync(settingsFilePath)) {
+      unlinkSync(settingsFilePath);
+    }
+  });
+
+   test("Verify can load settings.json", () => {
+     const serviceName = "Settings";
+     const expectedSettings = {"foo": "bar"};
+     writeFileSync(settingsFilePath, JSON.stringify(expectedSettings));
+
+     const actualSettings = helpers.readSettings(serviceName, settingsFilePath);
+
+     expect(actualSettings).toEqual(expectedSettings);
+   });
+
+   test("Verify cannot load settings.json because it does not exist", () => {
+     //eslint-disable-next-line no-console
```

Comment:
This is fine. I have no-console on for the main part of the code to ensure I don't accidentally use console.log in the actual app code.

Reply:
I'm open to either option. I typically use `next-line` if I'm only using it in a few spots in tests; otherwise I disable the whole file.

Reply:
Doesn't matter to me, this is fine. It is really on to force me to use the verbose/info/warn logging stuff in the main code.